op-node/eth/account_proof.go

```diff
@@ -82,40 +64,3 @@ func (res *AccountResult) Verify(stateRoot common.Hash) error {
 	}
 	return err
 }
-
-// BlockToBatch converts a L2 block to batch-data.
-// Invalid L2 blocks may return an error.
```

Comment:
Is this no longer used? Looking for where it was moved to

Reply:
Forgot to list this in the PR description, sorry. That's indeed unused code. The channel encoding code in the `derive` package has its own implementation of this.